doc/build-unix.md

```diff
@@ -82,7 +63,7 @@ install necessary parts of boost:
 BerkeleyDB is required for the wallet. db4.8 packages are available [here](https://launchpad.net/~bitcoin/+archive/bitcoin).
 You can add the repository and install using the following commands:
 
-    sudo add-apt-repository ppa:bitcoin/bitcoin
+    sudo add-apt-repository ppa:ElementsProject/elements
```

Comment:
We can make one if it doesn't already exist!  Once I finish with signed builds, we most likely will.